src/tests/ftest/deployment/server_rank_failure.yaml

```diff
@@ -68,11 +69,20 @@ ior_wo_rf:
   dfs_dir_oclass: SX
 ior_with_rp:
   <<: *ior_base
-  block_size: 150G
+  # Total size is block_size * ppn * number of clients. Because we frequently change the number of
+  # clients while debugging on Aurora, we may have to adjust block_size to make sure that IOR
+  # finishes before sw_deadline. (Smaller block_size for larger number of clients.) Use the second
+  # commented line for Aurora.
```

Comment:
Why use a stonewall if we want IOR to always finish before the stonewall?

Reply:
We want IOR to always finish before the stonewall during the normal condition with no failure. However, if there is a failure (rank stopped/engine killed), IOR may get stuck so we're using the stonewall.

Reply:
Even with a stonewall, IOR could hang indefinitely on some error conditions. In fact, we've been seeing exactly that lately on Aurora.
What you want instead is `ior_timeout`

Reply:
hmm, I noticed that setting `/run/ior/ior_timeout` in the test yaml will set `-D` in the IOR command if I'm using `run_ior_with_pool()`. Otherwise, the value will not be used anywhere. Since I'm not using `run_ior_with_pool()`, I need to create my own `job_manager` and manually set the timeout value to it.

Reply:
I don't understand how that happens.
The job manager timeout is set here
https://github.com/daos-stack/daos/blob/f825add501bef1c0961138aaa2ebdccab653f2e7/src/tests/ftest/util/ior_test_base.py#L147